docs/authentication/overview.mdx

```diff
@@ -1,5 +1,64 @@
+
+During login or registration, users will enter their email address to receive an email message with a link that can be clicked and complete the authentication process.
+
+[Multifactor](/docs/custom-flows/mfa)
+
+By enabling multifactor authentication (MFA), often referred to as two-factor authentication or 2FA, you can encourage or require your users to perform a second verification check during sign-in.
+
+[Social Login Providers](/docs/authentication/social-connections/overview)
+
+Clerk offers a list of Social Login providers that can be used during sign-up and sign-in. There will be a button on the sign-in and sign-up pages with the provider's logo and the appropriate text for each of the selected providers.
+
+Don't see the provider you need? Request others [here](https://clerk.com/support); we can usually add them within a week.
+
+## Frameworks
+
+Integrate Clerk with popular frameworks to get user management and authentication for your application in minutes.
+
+<div className="container mx-auto my-4">
+    <div className="grid grid-cols-1 gap-6 md:grid-cols-2">
+      <FrameworkCards title="React" description="Get started installing and initializing Clerk in a new Create React App." link="/docs/quickstarts/react" cta="Get Started" icon="/docs/images/logos/react.svg" />
+
+      <FrameworkCards title="Next.js" description="Easily add secure, beautiful, and fast authentication to Next.js with Clerk." link="/docs/quickstarts/nextjs" cta="Get Started" icon="/docs/images/logos/nextjs.svg" iconWidth="40" />
+
+      <FrameworkCards title="JavaScript" description="Add authentication and user management to your JavaScript application." link="/docs/quickstarts/javascript" cta="Get started" hideArrow={true} icon="/docs/images/logos/javascript.svg" iconWidth="30" />
+
+      <FrameworkCards title="Remix" description="Easily add secure, edge- and SSR-friendly authentication to Remix with Clerk." link="/docs/quickstarts/remix" cta="Get Started" icon="/docs/images/logos/remix.svg" />
+
+      <FrameworkCards title="Gatsby" description="Integrate user management and authentication into your Gatsby application." link="/docs/quickstarts/gatsby" cta="Get Started" hideArrow={true} icon="/docs/images/logos/gatsby.svg" />
+
+      <FrameworkCards title="RedwoodJS" description="Grow your RedwoodJS application with Clerk user management and authentication." link="/docs/quickstarts/redwood" cta="Get Started" icon="/docs/images/logos/redwood.svg" />
+
+      <FrameworkCards title="Expo / React Native" description="Use Clerk with Expo to authenticate users in your React Native application." link="/docs/quickstarts/expo" cta="Get Started" icon="/docs/images/logos/expo.svg" />
+
+      <FrameworkCards title="Fastify" description="Learn about installing and initializing Clerk in a new Fastify application." link="/docs/quickstarts/fastify" cta="Get Started" icon="/docs/images/logos/fastify.svg" iconWidth="40" />
+    </div>
+  </div>
```

Comment:
Also, not convinced that this is necessary either.

Reply:
I think I'd feel better about it if this grid of support framework cards was shared 🤔